blank-canvas-blocks/sass/blocks/_file.scss

```
@@ -0,0 +1,17 @@
+// Needed until https://github.com/WordPress/gutenberg/issues/27760 is fixed
```

Minor suggestion but could make our lives easier in the future:

```suggestion
// TODO: Remove when https://github.com/WordPress/gutenberg/issues/27760 is fixed
```